app/dashboard/templates/ftux/onboard.html

```diff
@@ -0,0 +1,69 @@
+{% comment %}
+  Copyright (C) 2018 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+
+{% load i18n static %}
+<!DOCTYPE html>
+<html>
+  <head>
+    {% include 'shared/head.html' %}
+    {% include 'shared/cards.html' %}
+    <link rel="stylesheet" href={% static "v2/css/onboard.css" %}>
+  </head>
+  <body>
+    {% include 'shared/tag_manager_2.html' %}
+    <div class="container-fluid header dash">
+      {% include 'shared/nav.html' %}
+    </div>
+    <div class="row">
+      <div class="col-12 offset-lg-2 col-lg-8 offset-xl-3 col-xl-6">
+        <form id="onboarding">
+          <div class="state">
+            {% for step in steps %}
+              <span class="step-state"></span>
+            {% endfor %}
+          </div>
+          {% for step in steps %}
+            <div id="step-{{ forloop.counter }}" link="{{ step }}" class="step">
+              {% include 'ftux/'|add:step|add:'.html' %}
+            </div>
+          {% endfor %}
+          <div class="controls">
+            <p id="prev-btn" class= ""onclick="changeStep(-1)">{% trans "Previous" %}</p>
+            <p id="next-btn" onclick="changeStep(1)">{% trans "Next" %}</p>
+          </div>
+        </form>
+      </div>
+    </div>
+    {% include 'shared/analytics.html' %}
+    {% include 'shared/footer_scripts.html' %}
+    {% include 'shared/rollbar.html' %}
+    {% include 'shared/messages.html' %}
+  </body>
+  {% csrf_token %}
+  <script type="text/javascript">
+    var csrftoken = jQuery("[name=csrfmiddlewaretoken]").val();
+  </script>
+  <script>
+    var steps = [];
+    var flow = "{{flow}}"
```

Comment:
rabble rabble 😁

Reply:
It helps with the URL redirection :P

Reply:
haha - I know.  I wasn't talking about the flow and step args exactly xD `{{flow}}` 😁